src/ensembl/src/content/home/components/homepage-species-bar/HomepageSpeciesBar.tsx

```diff
@@ -52,7 +53,7 @@ const HomepageSpeciesBar = (props: Props) => {
     );
   } else {
     const speciesItems = props.species.map((species, index) => (
-      <SimpleSelectedSpecies key={index} species={species} />
+      <SelectedSpecies key={index} species={species} onClick={noop} />
```

Comment:
Is this right? Are selected species on the home page really light blue meaning clickable? Isn't it better to describe them as all being active?

Reply:
That's done